### PR TITLE
Update enterprise licensing wording for SNOW

### PR DESCRIPTION
**Pull request by harrisonmeister**

### CODE REVIEW

**Review by robpearson — APPROVED** [review]
This aligns with the wording that we're using in product marketing. We are shifting to use the "subscription" wording as well. We've added a docs "landing page" in a separate PR(#1649).

File: docs/approvals/service-now/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ position: 10
 ---
 
 :::warning
-The ServiceNow Integration feature is available from Octopus **2022.3** onwards and requires an [enterprise license](https://octopus.com/enterprise).
+The ServiceNow Integration feature is available from Octopus **2022.3** onwards and requires an [enterprise subscription](https://octopus.com/enterprise).
```
Comment (robpearson) [inline docs/approvals/service-now/index.md:8]: Can this be updated to point to the pricing page? The enterprise page doesn't talk about the enterprise tier at all yet as it's a soft launch.

Comment (robpearson) [reply docs/approvals/service-now/index.md:8]: i.e. `https://octopus.com/pricing`

**Review by MJRichardson — APPROVED** [review]

File: docs/approvals/service-now/index.md (hunk 1 above)
Comment (harrisonmeister) [reply docs/approvals/service-now/index.md:8]:
```suggestion
The ServiceNow Integration feature is available from Octopus **2022.3** onwards and requires an [enterprise subscription](https://octopus.com/pricing).
```

Comment (harrisonmeister) [reply docs/approvals/service-now/index.md]: Done, thanks @robpearson

**Pull request merged**